js/ui/responsive_box.js

```diff
@@ -475,6 +480,10 @@ var ResponsiveBox = CollectionWidget.inherit({
         }, { _layoutStrategy: this.option("_layoutStrategy") });
     },
 
+    _needApplyAutoBaseSize: function(item) {
+        return !item.baseSize && (!item.minSize || item.minSize === "auto") && (!item.maxSize || item.maxSize === "auto");
```

Comment:
There is no "auto" value for minWidth/minHeight - default value is 0
There is no "auto" value for maxWidth/maxHeight - default value is none
https://www.w3schools.com/cssref/pr_dim_max-height.asp